Bottom: 19b7cd20e9a755de1e4bdae35871abe2daa4650d
Top:    bb3ee773e070c754fa652128180fa80e9875a2ee
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-09 12:23:48 +0100

Build with asserts for polygons


---

diff --git a/src/polygon.c b/src/polygon.c
index 3b302c3..751c3dc 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -73,6 +73,7 @@ dicer output is used for HIDs which cannot render things with holes
 /* special polygon editing routines
  */
 
+#undef NDEBUG
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
diff --git a/src/polygon1.c b/src/polygon1.c
index 5529aa7..3c6847b 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,6 +39,7 @@
       are marked
 */
 
+#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
@@ -103,7 +104,7 @@ if (UNLIKELY (((ptr) = malloc(sizeof(type))) == NULL)) \
 #undef DEBUG_JUMP
 #undef DEBUG_GATHER
 #undef DEBUG_ANGLE
-#undef DEBUG
+#define DEBUG
 #ifdef DEBUG
 #define DEBUGP(...) fprintf(stderr, ## __VA_ARGS__)
 #else
